undistro/zora
# UD-1955: Update trivy support to 0.57

## Description
Updated trivy support to 0.57.1

## Linked Issues

## How has this been tested?
- Deploy updated worker and operator images to cluster
- Deploy zora
- Check trivy download db job succeeds
- Check trivy plugin cronjob succeeds
- Check vulnerabilityreports

## Checklist
